iotdb/src/main/java/org/apache/iotdb/db/query/aggregation/impl/CountAggrFunc.java

```diff
@@ -52,9 +56,8 @@ public BatchData getResult() {
 
   @Override
   public void calculateValueFromPageHeader(PageHeader pageHeader) {
-    System.out.println("PageHeader>>>>>>>>>>>>" + pageHeader.getNumOfValues() + " " + pageHeader
-        .getMinTimestamp()
-        + "," + pageHeader.getMaxTimestamp());
+    LOGGER.debug("PageHeader>>>>>>>>>>>>num of rows:{}, minTimeStamp:{}, maxTimeStamp{}",
+        pageHeader.getNumOfValues(), pageHeader.getMinTimestamp(), pageHeader.getMaxTimestamp());
```

Comment:
If you are using return values as log parameter, wrap it with
`if (LOGGER.isDebugEnabled())`

Reply:
fixed